src/auth/auth.module.ts

```diff
@@ -7,7 +7,7 @@ import { AuthController } from "./auth.controller";
 import { JwtModule } from "@nestjs/jwt";
 import { JwtStrategy } from "./strategies/jwt.strategy";
 import { LdapStrategy } from "./strategies/ldap.strategy";
-import { ConfigModule, ConfigService } from "@nestjs/config";
+import { ConfigService } from "@nestjs/config";
```

Comment:
Why removing `ConfigModule`?
Also why keeping `ConfigService` when has been declared global in `app.module.ts`?

Reply:
Let\s make sure to document this. I see potential source of confusion, especially for new contributors or if it is a long time since last time we have touched the code